instr.ml

```diff
@@ -54,6 +54,22 @@ type binding =
   | Const of value
   | Mut of address
 
+let string_of_litteral : litteral -> string = function
+  | Nil -> "nil"
+  | Bool b -> string_of_bool b
+  | Int n -> string_of_int n
+
+let litteral_of_string : string -> litteral = function
+  | "nil" -> Nil
+  | "true" -> Bool true
+  | "false" -> Bool false
+  | n ->
+    try Int (int_of_string n) with _ ->
+      Printf.kprintf invalid_arg "litteral_of_string %S" n
```

Comment:
Explanations: `%S` prints a quoted quotes-and-newline-escaped string (it's good when you want to present unknown string values which may have whitespace etc.), `invalid_arg` is a standard function that just raises the `Invalid_argument exception`, and `kprintf` applies its first argument to the result of formatting the other arguments.